content/sensu-go/5.0/_index.md

```diff
@@ -49,43 +40,51 @@ The Sensu agent gives you visibility into everything you care about; the Sensu s
 - **Monitor containers, instances, applications, and on-premises infrastructure**
 
 Sensu is designed to monitor everything from the server closet to the cloud.
-[Install the Sensu agent][15] on the hosts you want to monitor, integrate with the Sensu API, or take advantage of [proxy entities][17] to monitor anything on your network.
+[Install the Sensu agent][15] on the hosts you want to monitor, integrate with the [Sensu API][23], or take advantage of [proxy entities][17] to monitor anything on your network.
```

Comment:
Not sure on the best phrasing, but it might be worth mentioning that proxy entities are for anything that you can't install the agent on for some reason. Networking devices are the best example of that, but you can use it for websites (create your own basic Pingdom), hardware devices running an unsupported operating systems or even devices that just don't have enough resources/internal policies don't allow the agent to run on it.

Reply:
This isn't new content to this PR, but feel free to open an issue to address this site-wide.